be/src/olap/rowset/segment_v2/binary_dict_page.h

```diff
@@ -32,6 +32,7 @@
 #include "runtime/mem_tracker.h"
 #include "gen_cpp/segment_v2.pb.h"
 #include "gutil/hash/string_hash.h"
+#include <util/owned_slice.h>
```

Comment:
Remove this line after `OwnedSlice` is moved into `slice.h`. In addition I want to point out that we use `#include <>` only for standard and third-party library.